ibm/resource_ibm_iam_api_key_test.go

```diff
@@ -0,0 +1,194 @@
+/**
+ * (C) Copyright IBM Corp. 2021.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ibm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/IBM/platform-services-go-sdk/iamidentityv1"
+)
+
+func TestAccIbmIamApiKeyBasic(t *testing.T) {
+	var conf iamidentityv1.APIKey
+	name := fmt.Sprintf("name_%d", acctest.RandIntRange(10, 100))
+	iamID := fmt.Sprintf("iam_id_%d", acctest.RandIntRange(10, 100))
```

Comment:
Can the iamID can be generated randomly?...I think it shd be a valid user ID for the testcases to execute